### DISCUSSION TITLE
Video Room Plugin

### DISCUSSION THREAD

**Discussion by ChiragLiveu** [flutterjanus/flutter_janus_client#129] [Q&A] [answered]
I have implemented the Video Room plugin in one of the Flutter demos. Can you help me with the below questions?
How can we set the video encoder bitrate and delay value for the video?
How can we check that it's using software encoders or Hardware encoders? 
Is there any way to switch between hardware and software encoder while streaming? 
I am also facing more packet loss while streaming any suggestions to resolve this?

I am new to Janus can you please help me with the above questions

Thank you

**Comment by shivanshtalwar0** [comment, marked as answer]
I think you should refer to Janus videoroom documentation for Janus-specific questions
